README.md

```diff
@@ -20,12 +20,11 @@ Requirements:
     Module: os
     Module: subprocess
     Module: argparse
+    Module: itertools
     Module: tempfile
     Module: shutil
```

Comment:
I like the idea to document the used dependencies, but all listed modules are default python modules that should always be shipped with python installations. Thus, this list is too detailed. If you are going to use non-default libraries like e.g. `pandas` or `seaborn` we can list them here AND have to add them to the `environment.yml` file